src/controllers/phone.controllers.ts

```diff
@@ -59,8 +59,21 @@ const discountedPhones = async (req: Request, res: Response) => {
   res.send(discountedProducts);
 };
 
+const getPhone = async (req: Request, res: Response) => {
+  const phoneId = req.params['phoneId'];
+
+  const getPhoneById = await sequelize.query(
+    `SELECT * FROM "PhoneDetails" WHERE "phoneId"='${phoneId}'`, {
+      type: QueryTypes.SELECT
+    }
+  );
```

Comment:
this part should be separated into service directory, but if you'll agreed upon this structure - got for it, just saying 😄

Reply:
Probably if we have some time left, we'll work on cleaning code a bit :)